src/foam/nanos/logger/AbstractLogger.js

```diff
@@ -84,7 +84,6 @@ return String.valueOf(obj);`
       javaCode:
       `StringBuilder str = sb.get();
 for ( Object n : args ) {
-  str.append(',');
```

Comment:
Why remove?

Reply:
@deejes this should either be moved to after str.append(formatArg(n)) and/or use a 'for index' loop and only append it if there are more args.

Reply:
@jlhughes I think what you're saying didn't used to be required because the timestamp was already output so a comma was needed before each argument no matter the position.